frameworks/elastic/tests/test_tls.py

```diff
@@ -2,15 +2,18 @@
 import shakedown
 
 import sdk_install
+import sdk_hosts
 import sdk_plan
 import sdk_utils
 
 from security import transport_encryption
 
 from tests import config
 
-pytestmark = pytest.mark.skipif(sdk_utils.is_open_dcos(),
-                                reason='Feature only supported in DC/OS EE')
+pytestmark = [pytest.mark.skipif(sdk_utils.is_open_dcos(),
+                                 reason="Feature only supported in DC/OS EE"),
+              pytest.mark.skipif(sdk_utils.dcos_version_less_than("1.10"),
+                                 reason="TLS tests require DC/OS 1.10 or higher")]
```

Comment:
I've seen a lot of other messages using `1.10+`. Not sure which you prefer, but it's not critical. (only change it if applicable and if there are other changes that need the tests to run again)

Reply:
I can change it, but I probably copied from [here](https://github.com/mesosphere/dcos-commons/blob/a6b8f0c1b61f37bae4e7dc071a0aff30a6cacbc6/frameworks/hdfs/tests/test_kerberos_auth.py#L25-L26) fwiw.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2349/commits/554730b7ecfadde27a7cd43be93408238688d641